app/controllers/companies_controller.rb

```diff
@@ -25,4 +24,13 @@ def update_companies_details
     flash[:notice] = 'Companies details successfully updated'
     redirect_to companies_url
   end
+
+  def validate_params
```

Comment:
```suggestion
  private
  
  def validate_index_params
```

Reply:
but I don't have a strong opinion about adding `index` to the name. Making method private is more important.

Reply:
It is ok with the name as well, will be more precise